src/views/shared/TextBox.js

```diff
@@ -52,8 +52,8 @@ function ($, Handlebars, BrowserFeatures, TextBox) {
           <span class="icon input-icon {{params.icon}}"></span>\
         {{/if}}\
       {{/if}}\
-      <input type="{{type}}" placeholder="{{placeholder}}" name="{{name}}" \
-        id="{{inputId}}" value="{{value}}" autocomplete="off"/>\
+      <input type="{{type}}" placeholder="{{placeholder}}" aria-label="{{placeholder}}"\
```

Comment:
aria-labelledby contains ids. I don't have visible elements that it can point to. So aria-labelledby won't work unless I add hidden <div> or <label>